parqueryd/rpc.py

```diff
@@ -111,8 +124,16 @@ def _rpc(*args, **kwargs):
             if name == 'groupby' and rep == '':
                 # this is the placeholder for an empty result from a groupby and needs to be explicitly caught
                 return None
+            elif not rep and last_except:
+                    parsed_exception = _parse_exception(last_except)
+                    if parsed_exception:
+                        self.logger.critical("No response from DQE, retries %s exceeded" % self.retries)
+                        raise parsed_exception  
+                    else: 
+                        raise RPCError("No response from DQE, retries %s exceeded" % self.retries)
```

Comment:
Maybe also create a separate exception for `RetriesExceeded`

Reply:
Oh right, sure.